### PR TITLE
add command `convert_usermode` #404

### PR DESCRIPTION
**Pull request by Krishna-Singhal**
Now users can give your phone number to enable user mode

### CODE REVIEW

**Comment by ZekXtreme** [comment]
Nice ..seems cool and useful

**Comment by Jigarvarma2005** [comment]
> Now users can give your phone number to enable user mode


> Now users can give your phone number to enable user mode

Now more easy to generate user session str and save automatically 🔥

**Comment by Krishna-Singhal** [comment]
> > Now users can give your phone number to enable user mode
> 
> Now more easy to generate user session str and save automatically 🔥

ah yes

File: userge/plugins/tools/system.py [hunk 1]
```diff
@@ -286,6 +289,131 @@ async def view_disabled_chats_(message: Message):
...
+            try:
+                if hasattr(generate_session, "two_step_code"):
+                    delattr(generate_session, "two_step_code")
+                setattr(generate_session, "phone_code", code)
+                if await generate_session():
+                    session_string = await generate_session.client.export_session_string()
+                    await msg.reply(
+                        "DONE! User Mode will be enabled after restart."
+                    )
+                    Config.HEROKU_APP.config()["HU_STRING_SESSION"] = session_string
```
Comment (rking32) [inline userge/plugins/tools/system.py:323]: what about non heroku users
```python
if Config.HEROKU_APP:
    # handle heroku user
else:
    # handle non-heroku user
```
maybe we can show env key and value to the user?

Comment (Krishna-Singhal) [reply userge/plugins/tools/system.py:323]: > what about non heroku users
> 
> ```python
> if Config.HEROKU_APP:
>     # handle heroku user
> else:
>     # handle non-heroku user
> ```
> 
> maybe we can show env key and value to the user?

hmm 🤔

**Comment by rking32** [comment]
NIce :)

**Pull request merged**